data/themes/darktable.css

```diff
@@ -1017,10 +1017,16 @@ entry
   margin: 8px 0px;
 }
 
+#left #lib-plugin-ui scrolledwindow entry /* This will avoid have too much height on entry item in clone manager without touching other parts of the UI */
```

Comment:
avoid have -> avoid having
entry item -> entry items ?

Reply:
Oups. Commit submitted for better English comment ! Thanks !